app/helpers/application_helper.rb

```diff
@@ -132,27 +132,17 @@ def no_hover_class(item)
   end
 
   # Check role based authorization for a UI task
-  def role_allows(options = {})
-    ApplicationHelper.role_allows_intern(options) rescue false
+  def role_allows(**options)
```

Comment:
@chrisarcand agreed . . . back then we were like "we don't need no stinkin' question marks!"